packages/deployer/src/commands.ts

```diff
@@ -29,9 +31,9 @@ export async function deploy(configPath, secretsPath, outputFilename, nonStop, n
   } else if (!validateMnemonic(secrets.MASTER_KEY_MNEMONIC)) {
     ora().fail('MASTER_KEY_MNEMONIC in your secrets.env file is not valid');
   }
-  delete secrets['AWS_ACCESS_KEY_ID'];
-  delete secrets['AWS_SECRET_KEY'];
-  fs.writeFileSync('secrets.json', JSON.stringify(secrets, null, 2));
+  const tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'airnode'));
```

Comment:
What if I run two `deploy`s in parallel?

Reply:
One will fail. Either you'll fail to create an S3 bucket or DynamoDB table for second time or Terraform will lock the provisioning (that's what the DynamoDB table is for) and won't let you run the recipes.